lib/statsd/instrument/batched_udp_sink.rb

```diff
@@ -6,11 +6,14 @@ module Instrument
     #   to become the new default in the next major release of this library.
     class BatchedUDPSink
       DEFAULT_FLUSH_INTERVAL = 1.0
-      MAX_PACKET_SIZE = 508
```

Comment:
I can't remember why it was set so low. The PR introducing this value doesn't discuss it. However I took this value from other statsd clients.